tests/broken_options_test.cpp

```diff
@@ -241,8 +243,166 @@ about
                                           description
   -t [ --tog ]                            some toggle
   -u [ --togg ]                           some other toggle
+)EXPECTED";
+
+        CHECK(actual.size() == expected.size());
+
+        std::size_t tmp = 0;
```

Comment:
Please remove the debugging code. Ideally, this test shouldn't change at all. (Except for the newline)